tendrl/notifier/notifier/utils/central_store_util.py

```diff
@@ -0,0 +1,69 @@
+from etcd import EtcdException
+from etcd import EtcdKeyNotFound
+from tendrl.commons.objects.alert import Alert
+from tendrl.commons.objects.cluster_alert import ClusterAlert
+from tendrl.commons.objects.node_alert import NodeAlert
+
+
+def read_key(key):
```

Comment:
wherever this is used, use the one defined in commons instead..
if not used remove this func